locales/en.yml

```diff
@@ -28,5 +28,37 @@ en:
       col_2: "Column 2"
       col_3: "Column 3"
       col_4: "Column 4"
+    spreadsheet:
+      no_records_found: "No records found"
+      showing_page: "Showing page {0} of {1} entries"
+      show: "Show"
+      entries: "entries"
+      about: "About"
+      prompts:
+        delete_selected_rows: "Are you sure to delete the selected rows?"
```

Comment:
Should this be "Are you sure **you want** to delete the selected rows?"

Reply:
Thanks @heymarkreeves good catch, I had just blindly copied the default values from the library.